### PR TITLE
scripts/netContain/scripts/{contivNet.sh,ovsInit.sh}: init OVS before doing anything else and load OVS kernel module before starting OVS services

### PR DESCRIPTION
**Pull request by dseevr**
Signed-off-by: Bill Robinson <dseevr@users.noreply.github.com>

### CODE REVIEW

File: scripts/netContain/scripts/contivNet.sh [hunk 1]
```diff
@@ -76,6 +76,12 @@ while getopts ":xmp:v:i:c:drl:o:" opt; do
 	esac
 done
 
+if [ $netplugin == true ]; then
+	echo "Initializing OVS"
+	/contiv/scripts/ovsInit.sh
+	echo "Initialized OVS"
+fi
+
```
Comment (rchirakk) [inline scripts/netContain/scripts/contivNet.sh]: if $cleanup == true, then the script shouldn't start ovs process

Comment (dseevr) [inline scripts/netContain/scripts/contivNet.sh]: Fixed

**Comment by rchirakk** [comment]
LGTM

**Comment by rhim** [comment]
@rchirakk let's merge this for 1.1.1.

**Pull request merged**